extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -905,6 +969,21 @@ export class AmpA4A extends AMP.BaseElement {
 
   /** @override */
   layoutCallback() {
+    if (this.isRefreshing) {
```

Comment:
I believe that in either of those cases isRefresh will be set to false in tearDownSlot, which will cause the refresh function to short-circuit on line 804.